frontend/controller/actions/group.js

```diff
@@ -262,15 +263,15 @@ export default (sbp('sbp/selectors/register', {
     // JOINING_GROUP, so that we process leave actions and don't interfere
     // with the leaving process (otherwise, the side-effects will prevent
     // us from fully leaving).
-    sbp('chelonia/contract/wait', [params.originatingContractID, params.contractID])
+    await sbp('chelonia/contract/wait', [params.originatingContractID, params.contractID])
```

Comment:
Looks like you caught a bug from a previous PR that went unnoticed?

Reply:
Right. As it was it did nothing.